docs/USER_GUIDE.md

```diff
@@ -0,0 +1,323 @@
+
+Those evironmental setting are dictated by Docker file which has instructions of how to build the environment (container). You can customize it on your will. 
+
+After running the docker file, Docker will create a corresponding container that is the environment you need to run applications. The last and important property is that Docker allows different containers operating simultaneously without affecting to other containers.
+```
+
+Because of those properties, when a user want to take advantage of server's computing capability, he no more worries about problems caused by different environments and he can focus on his project.  
+
+Besides integrating Docker, our team try to make this system more usable so that we add additional functions to it. For example:  
+
+1. `User can customize their required resources with CLI.`  
+2. `The host maintainer can supervise the usage of server easily.`  
+
+Now, let's start the intructions of AIVC-Server-Booking system.
+
+---
+
+## **`2. Before Booking......`**
+
+Before booking AIVC server, there are somthing you should know ......
+
+1. Ask host maintainer for a new account first. Every account has default limitation of resources. If you want more, you can go to discuss with the host maintainer.
+
+2. You will be forbidden to access server if your used space is out of range. You must sort out your memory space before using server.
+
+3. Every container is removed after it run out of time to release the computing resources being utilized.
+
+4. There are three main directories in each container,  which those directories will be volumed to corresponding ones in host. Let's take a look for their purposes :  
+    * `backup_dir` :  Aims to store the configs customized by the user. Next time, when you want to book server again, the booking system will build the container based on your configs, which are already backed up in `backup_dir`, to build the customization.  
+    When you want to backup files or directories, you should write down you needs in `backup.yaml` and follow the form shown below :  
+
+        ```yaml  
+        # Under the Backup
+        Dir: 
+          - - < Backup's dir_path>    # Backup is a folder in host
+            - < container's dir_path> # must be absolute path  
+        
+        File: 
+          - - < Backup's file_path>    # Backup is a folder in host
+            - < container's file_path> # must be absolute path
+        ```  
+
+        For example :
+
+        ```yaml  
+        Dir: 
+        - - .default/.pyenv 
+          - /root/.pyenv 
+        File: 
+        - - .zshrc 
+          - /root/.zshrc 
+        ```
+
+    * `work_dir` :  Store user's projects and  personal dataset. Because booking system will automatically delete container with empty work_dir, you should ensure the work_dir has projects in itself to avoid your container from being removed.
```

Comment:
> Because booking system will automatically delete container with empty work_dir, you should ensure the work_dir has projects in itself to avoid your container from being removed.
 
This is terribly wrong!!